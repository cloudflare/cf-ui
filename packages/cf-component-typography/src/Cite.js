import React from 'react';

import PropTypes from 'prop-types';

class Cite extends React.Component {
  render() {
    return (
      <cite className="cf-cite">
        {this.props.children}
      </cite>
    );
  }
}

Cite.propTypes = {
  children: PropTypes.node
};

export default Cite;
