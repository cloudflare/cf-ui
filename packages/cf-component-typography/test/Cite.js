import React from 'react';
import renderer from 'react-test-renderer';
import Cite from '../src/Cite';

test('should render', () => {
  const component = renderer.create(<Cite>Cite</Cite>);
  expect(component.toJSON()).toMatchSnapshot();
});
