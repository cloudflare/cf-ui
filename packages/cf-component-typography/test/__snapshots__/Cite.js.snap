// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<cite
  className="cf-cite"
>
  Cite
</cite>
`;
